packages/ui/components/slideshow/index.tsx

```diff
@@ -0,0 +1,73 @@
+// This file is over 50 lines and needs to be split up
+
+import React, { Component } from 'react';
+import { View, Image, ScrollView, Dimensions, Animated } from 'react-native';
+
+import style from 'theme/slideshow'
+
+const { width } = Dimensions.get('window');
+
+const sixtyFpsInMs = 16
+
+interface Props {
+  views: any
+}
+
+export default class Slideshow extends Component<Props> {
+
+  getBaseScroll = new Animated.Value(0)
+
+  displayContent () {
+    return this.props.views.map((source, i) => {
+      return (
+        <View key={i} style={[{ width }, style.content]}>
+          {source}
+        </View>
+      );
+    })
+  }
+
+  createIndictor () {
```

Comment:
Spelling: `createIndicator()` 

Also this is a render method, not a creation method. Any method that returns JSX - like the standard `render` - should start with `render` as in `renderIndicator()`